apps/api/src/common/constants/notifications.ts

```diff
@@ -18,6 +18,7 @@ export const ChannelType = {
   SMS_KAPSYSTEM: 8,
   PUSH_SNS: 9,
   VC_TWILIO: 10,
+  AWS_SES: 12,
```

Comment:
We already have a PR which uses channel type 11, PR 306, SNS SES Provider

Reply:
Since this PR is 305, opened earlier than 306, let's follow the correct order to avoid migrations going of sync. This PR will be merged first then 306. Make AWS_SES as 11.
Other PR will be updated to use AWS_SNS as 12 (https://github.com/OsmosysSoftware/osmo-x/pull/306#discussion_r1721501344).

Reply:
Updated. Made channeltype as 11